.gitlab-ci.yml

```diff
@@ -100,6 +100,36 @@ SonarQube:
     INTERNAL_NETWORK: "true"
     GIT_DEPTH: 0
 
+Manifests:
+  stage: test
+  extends: .terraform
+  script:
+    - schutzbot/deploy.sh
+    - sudo schutzbot/manifest_tests.sh
```

Comment:
This is my fault, because I suggested moving it away from `test/cases` with the reason being that this is not an osbuild test, but an external one, very much like currently the composer tests are also not in `test/cases`. We could package the manifest db data and scripts up, install them and then just call one entry point executable very much as we do for the existing composer tests.

Reply:
Fair. Thanks for the context.